### ISSUE TITLE
Is the tutorial ok?

### ISSUE THREAD

**Issue by customcommander** [lit/lit.dev#588]
I was expecting the checkbox to hide completed tasks.
https://lit.dev/tutorial/#08-finishing-touches

Google Chrome Version 95.0.4638.69 (Official Build) (x86_64)

![Screenshot 2021-11-12 at 10 43 44](https://user-images.githubusercontent.com/681975/141458234-85849426-090c-4f6a-8754-ffdf2534da75.png)

**Comment by AndrewJakubowicz** [comment]
Thanks for the issue! I've transferred it to the lit.dev repository.
I checked and I think it works after you solve the left hand side instructions.

My reproducing steps attached:

![final-tutorial-repro-litdev](https://user-images.githubusercontent.com/15080861/141530477-92b98b27-53e3-4dfe-847f-736635688f5f.gif)

Let me know if it still doesn't work after it's solved. Thank you!

cc/ @arthurevans - Maybe it's worth have a final tutorial panel that requires no work and shows only the completed code?
